src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -309,6 +309,8 @@ def test_get_pin_results_pin_information(multi_instrument_session):
     assert channels == fully_qualified_channels
 
 
+# TODO(jfitzger): make enum and HistoryRAMCycleInformation access consistent with the rest of the tests
+# and remove unnecessary imports.  Blocked by GitHub issue# 1426.
```

Comment:
Move
```
from nidigital.enums import PinState
from nidigital.history_ram_cycle_information import HistoryRAMCycleInformation
```
to inside the repr() test method. That way we don't showcase the `from x import y` style at the top of the test module.

Reply:
Done.